packages/shared/src/modules/quiz/libs/enums/quiz-validation-message.enum.ts

```diff
@@ -0,0 +1,6 @@
+const QuizValidationMessage = {
+	NUMBER_ID_REQUIRED: "Required answer IDs must be numbers.",
```

Comment:
This is not a very user-friendly message

Reply:
Changed to that one:
```
NUMBER_ID_REQUIRED: "Please enter only numeric values for answers."
```
Would that be ok or should I get rid of showing message for that check at all?

Reply:
We can have not very detailed message for a user, like "Invalid request" or something else